Comparator.dcm

```diff
@@ -181,9 +181,9 @@ F http://ww1.microchip.com/downloads/en/DeviceDoc/DS20002143E.pdf
 $ENDCMP
 #
 $CMP MCP6567
-D Dual 1.8V Low-Power Push-Pull Output Comparator, SOIC-8/MSOP-8
+D Dual 1.8V Low-Power Open-Drain Output Comparator, SOIC-8/MSOP-8
 K dual comparator
-F http://www.farnell.com/datasheets/2133492.pdf
+F http://ww1.microchip.com/downloads/en/DeviceDoc/DS20002143E.pdf
```

Comment:
That link is dead right now.
The Datasheet for `MCP6562` also does not work. Redirect error.
I will revisit the Microchip MCP656[2,7] later.

Reply:
All our Microchip DS links are dead.
They include the device name in the URL now and couldn't be bothered to redirect the old ones...

Reply:
I did start a datasheet downloader to get all datasheets. There was a protoype with gigabytes of data, but did not bother to save the VM I used for that :/

I did abandon it since the legal status of providing a mirror was not clear. I guess its time to revisit that project. Most likely without a public interface.